lib/ares.rb

```diff
@@ -7,6 +7,10 @@
 require 'ico-validator'
 
 module Ares
+  cattr_accessor :timeout do
```

Comment:
But in this way (constant) it would not be possible to change it outside. Now, it works like:
```ruby
    Ares.timeout = 4
    if Ares.standard(ico: data.ico)
```